library/pcs_cluster.py

```diff
@@ -91,6 +95,16 @@
     state: 'present'
   run_once: True
 
+- name: Create cluster with redundant corosync links and transport options
```

Comment:
(low priority) "- name: Create cluster with redundant corosync links and transport and link options"